src/reformat.c

```diff
@@ -407,15 +407,15 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image, avifRGBIm
                     // in the following unormU and unormV grid as follows:
                     //
                     // unorm[0][0] = closest         ( weights: bilinear: 9/16, nearest: 1 )
-                    // unorm[1][0] = adjacent row    ( weights: bilinear: 3/16, nearest: 0 )
-                    // unorm[0][1] = adjacent col    ( weights: bilinear: 3/16, nearest: 0 )
+                    // unorm[1][0] = adjacent col    ( weights: bilinear: 3/16, nearest: 0 )
+                    // unorm[0][1] = adjacent row    ( weights: bilinear: 3/16, nearest: 0 )
```

Comment:
I changed the comments for unorm[1][0] and unorm[0][1] to match the code below:

                        unormU[1][0] = ptrU8[(uvI * yuvChannelBytes) + uAdjCol];
                        unormV[1][0] = ptrV8[(uvI * yuvChannelBytes) + vAdjCol];
                        unormU[0][1] = ptrU8[(uvI * yuvChannelBytes) + uAdjRow];
                        unormV[0][1] = ptrV8[(uvI * yuvChannelBytes) + vAdjRow];

I.e., I assume the code is correct and changed the comments to match.